Src/CSharpier/CodeFormatter.cs

```diff
@@ -173,11 +173,11 @@ private string PrintDocTree(Doc document, string indent)
                         indent + "    "
                     ) + ")";
                 case LeadingComment leadingComment:
-                    return indent + "LeadingComment(" + leadingComment.Comment + ", CommentType." + (leadingComment.Type == CommentType.SingleLine
+                    return indent + "LeadingComment(\"" + leadingComment.Comment + "\", CommentType." + (leadingComment.Type == CommentType.SingleLine
```

Comment:
You might find these easier to do with string interpolations.

Also, is this meant to output valid C# code that you can paste into unit tests or something? If not, you might find it easier to use single quotes rather than double; that way you don't have to escape the quotes all the time.